perun-utils/password-manager/perun.passwordManager

```diff
@@ -136,8 +136,8 @@ function reserve() {
 		fi
 
 		# If empty password was supplied then set random one
-		if [ -z "${PASSWORD}" ]; then
-			logger "Perun-PasswordReserve: empty password provided for $LOGINNAMESPACE:$USERLOGIN in $REALM."
+		if [ ${#PASSWORD} -lt 8 ]; then
+			logger "Perun-PasswordReserve: At least 8 characters long password needs to be provided for $LOGINNAMESPACE:$USERLOGIN in $REALM."
```

Comment:
```suggestion
			logger "Perun-PasswordReserve: password shorter than 8 characters provided for $LOGINNAMESPACE:$USERLOGIN in $REALM."
```
This would match the tone of the previous log, however just a nitpick

Reply:
Agree, we should stay consistent.